src/test/java/org/folio/edge/rtac/RtacHandlerTest.java

```diff
@@ -0,0 +1,145 @@
+package org.folio.edge.rtac;
+
+import static org.apache.http.HttpStatus.SC_NOT_ACCEPTABLE;
+import static org.apache.http.HttpStatus.SC_OK;
+import static org.folio.edge.core.Constants.APPLICATION_JSON;
+import static org.folio.edge.core.Constants.APPLICATION_XML;
+import static org.folio.edge.core.Constants.TEXT_PLAIN;
+import static org.junit.Assert.assertEquals;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import io.restassured.RestAssured;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.stream.Collectors;
+import org.apache.commons.lang3.ArrayUtils;
+import org.apache.http.HttpHeaders;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.folio.edge.rtac.model.Instances;
+import org.folio.edge.rtac.utils.RtacMockOkapi;
+import org.junit.Test;
+
+public class RtacHandlerTest extends MainVerticleTest {
```

Comment:
Please add a test for the expected behaviour if both media types are acceptable to the client

Reply:
Done